openreview/conference/templates/programchairWebfield.js

```diff
@@ -959,31 +959,54 @@ var displayStatsAndConfiguration = function(conferenceStats) {
 
   // Timeline
   var invitationMap = conferenceStatusData.invitationMap;
-
   html += '<div class="col-md-8 col-xs-12">'
   html += '<h4>Timeline:</h4><ul style="padding-left: 15px">';
-  html += renderInvitation(invitationMap, SUBMISSION_ID, 'Paper Submissions')
-  html += renderInvitation(invitationMap, REVIEWERS_ID + '/-/' + BID_NAME, 'Reviewers Bidding')
+  var notDatedElements = [];
+  var datedElements = [];
+  var pushToDatedArrays = function(invitationMap, id, name, dated, notDated){
+    var currContent = renderInvitation(invitationMap, id, name);
+    if (!currContent) return;
+
+    var currDate = invitationMap[id].duedate;
+    if (currDate){
```

Comment:
Curly braces should have spaces around them and else should be on the same line as the closing brace.